content/pt-br/docs/tasks/configure-pod-container/migrate-from-psp.md

```diff
@@ -0,0 +1,389 @@
+essas mudanças com segurança. 
+
+{{< /caution >}}
+
+- `.spec.runAsGroup`
+- `.spec.supplementalGroups`
+- `.spec.fsGroup`
+
+Os campos de mutação restantes são necessários para suportar adequadamente os padrões 
+de segurança de Pod, e precisarão ser tratados caso a caso mais tarde:
+
+- `.spec.requiredDropCapabilities` - Necessário para derrubar `ALL` para o perfil restrito.
+- `.spec.seLinux` - (Apenas mutação com a regra `MustRunAs`) necessário para fazer cumprir 
+  os requisitos de SELinux da `Baseline` e Perfis restritos.
+- `.spec.runAsUser` - (Não mutante com a regra `RunAsAny`) necessário para aplicar 
+  `RunAsNonRoot` para o perfil restrito.
+- `.spec.allowPrivilegeEscalation` - (Se somente mutante estiver como `false`) 
+  necessário para o perfil restrito.
+
+### 2.c. Implantando os PSPs atualizados {#psp-update-rollout}
+
+Em seguida, você pode implantar as políticas alteradas ao seu cluster. 
+Você deve prosseguir com cautela, a remoção das opções de mutação pode resultar 
+em cargas de trabalho com perda da configuração necessária.
+
+Para cada atualização `PodSecurityPolicy`:
+
+1. Identifique Pods em execução sob o PSP original. Isso pode ser feito usando 
+a anotação `kubernetes.io/psp`. 
+Por exemplo, usando o kubectl:
+   ```sh
+   PSP_NAME="original" # Defina o nome do PSP que você está checando
+   kubectl get pods --all-namespaces -o jsonpath="{range .items[?(@.metadata.annotations.kubernetes\.io\/psp=='$PSP_NAME')]}{.metadata.namespace} {.metadata.name}{'\n'}{end}"
+   ```
+2. Compare esses Pods em execução com a especificação original do Pod, 
+   para determinar se a `PodSecurityPolicy` modificou o Pod. Para pods criados pelo 
+   [recurso de cargas de trabalho](/docs/concepts/workloads/controllers/)
+   você pode comparar o pod com o `PodTemplate` no controlador de recurso. 
+   Se alguma alteração for identificada, o Pod original ou `PodTemplate` 
+   deve ser atualizado com a configuração desejada.
+   Os campos para revisar são:
+   - `.metadata.annotations['container.apparmor.security.beta.kubernetes.io/*']` 
+     (Substitua * pelo nome de cada contêiner)
+   - `.spec.runtimeClassName`
+   - `.spec.securityContext.fsGroup`
+   - `.spec.securityContext.seccompProfile`
+   - `.spec.securityContext.seLinuxOptions`
+   - `.spec.securityContext.supplementalGroups`
+   - Em contêneres, sob `.spec.containers[*]` e `.spec.initContainers[*]`:
+       - `.securityContext.allowPrivilegeEscalation`
+       - `.securityContext.capabilities.add`
+       - `.securityContext.capabilities.drop`
+       - `.securityContext.readOnlyRootFilesystem`
+       - `.securityContext.runAsGroup`
+       - `.securityContext.runAsNonRoot`
+       - `.securityContext.runAsUser`
+       - `.securityContext.seccompProfile`
+       - `.securityContext.seLinuxOptions`
+3. Crie a nova `PodSecurityPolicies`. Se quaisquer `Roles` ou `ClusterRoles` estão 
+   permitindo `use` em todas as PSPs, isso poderia ocasionar que um novo PSPs 
```

Comment:
The term `use` can be confused in Portuguese. Try rearranging like this "permitindo em todas as PSPs o `use`"